src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -170,41 +170,18 @@ private[spark] class KubernetesSuite extends FunSuite with BeforeAndAfterAll wit
     }
   }
 
-  test("Run PageRank using remote data file") {
+  test("Test extraJVMProprties being present on Driver") {
     sparkAppConf
-      .set("spark.kubernetes.mountDependencies.filesDownloadDir",
-        CONTAINER_LOCAL_FILE_DOWNLOAD_PATH)
-      .set("spark.files", REMOTE_PAGE_RANK_DATA_FILE)
-    runSparkPageRankAndVerifyCompletion(
-      appArgs = Array(CONTAINER_LOCAL_DOWNLOADED_PAGE_RANK_DATA_FILE))
+      .set("spark.driver.extraJavaOptions", "-Dspark.test.foo=spark.test.bar")
+    runSparkJVMCheckAndVerifyCompletion(
+      expectedJVMValue=Seq("spark.test.foo","spark.test.foo"))
```

Comment:
Think we probably want something more specific, e.g. "The property spark.test.foo was properly set to spark.test.bar"

Reply:
The result in the test is printing as a tuple (spark.test.foo,spark.test.bar) to show k-v pair. Will that suffice?